config/default-spinnaker-local.yml

```diff
@@ -127,9 +127,9 @@ providers:
       name: my-docker-registry
       address: ${SPINNAKER_DOCKER_REGISTRY:https://index.docker.io/}
       repository: ${SPINNAKER_DOCKER_REPOSITORY:library/nginx}
-      username: ${SPINNAKER_DOCKER_USERNAME}
+      username: ${SPINNAKER_DOCKER_USERNAME:}
       # A path to a plain text file containing the user's password
-      passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE}
+      passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE:}
```

Comment:
If I don't have the colons in 'spinnaker-local', I get this in 'localhost:8084/resolvedEnv':
> "providers.dockerRegistry.primaryCredentials.passwordFile": "Exception occurred: Could not resolve placeholder 'SPINNAKER_DOCKER_PASSWORD_FILE' in string value \"${SPINNAKER_DOCKER_PASSWORD_FILE}\""

If I also don't have the colons in 'clouddriver.yml', I get this exception when I check the clouddriver log:

> java.io.FileNotFoundException: ${providers.dockerRegistry.primaryCredentials.passwordFile} (No such file or directory)

There's many reasons the config could've failed in #1058 and I'm not convinced the colon was a problem. From what I can see the colon is used in many cases throughout these files